src/modules/Game.class.js

```diff
@@ -1,68 +1,344 @@
 'use strict';
 
-/**
- * This class represents the game.
- * Now it has a basic structure, that is needed for testing.
- * Feel free to add more props and methods if needed.
- */
+// const rows = document.querySelectorAll('tr');
+
 class Game {
-  /**
-   * Creates a new game instance.
-   *
-   * @param {number[][]} initialState
-   * The initial state of the board.
-   * @default
-   * [[0, 0, 0, 0],
-   *  [0, 0, 0, 0],
-   *  [0, 0, 0, 0],
-   *  [0, 0, 0, 0]]
-   *
-   * If passed, the board will be initialized with the provided
-   * initial state.
-   */
-  constructor(initialState) {
-    // eslint-disable-next-line no-console
-    console.log(initialState);
+  constructor(
+    initialState = [
+      [0, 0, 0, 0],
+      [0, 0, 0, 0],
+      [0, 0, 0, 0],
+      [0, 0, 0, 0],
+    ],
+  ) {
+    this.board = initialState.map((row) => [...row]);
+    this.score = 0;
+    this.status = 'idle';
+    this.initialState = initialState;
+  }
+
+  getState() {
+    return this.board;
+  }
+
+  getScore() {
+    return this.score;
+  }
+
+  getStatus() {
+    return this.status;
+  }
+
+  start() {
+    this.status = 'playing';
+    this.newCell();
+    this.newCell();
+  }
+
+  restart() {
+    this.status = 'idle';
+    this.score = 0;
+
+    this.board = this.initialState.map((row) => [...row]);
```

Comment:
When restarting the game, you're resetting the board to the `initialState`. This might lead to unexpected behavior if the `initialState` is changed after the game starts. It would be safer to store a deep clone of the `initialState` at construction and reset to that clone.